modules/core/arrow-test/src/main/kotlin/arrow/test/laws/BitraverseLaws.kt

```diff
@@ -0,0 +1,28 @@
+package arrow.test.laws
+
+import arrow.Kind
+import arrow.Kind2
+import arrow.core.ForId
+import arrow.core.Id
+import arrow.core.extensions.id.applicative.applicative
+import arrow.core.extensions.id.comonad.extract
+import arrow.test.generators.functionAToB
+import arrow.test.generators.intSmall
+import arrow.typeclasses.Bitraverse
+import arrow.typeclasses.Eq
+import io.kotlintest.properties.Gen
+import io.kotlintest.properties.forAll
+
+object BitraverseLaws {
+  fun <F> laws(BT: Bitraverse<F>, cf: (Int) -> Kind2<F, Int, Int>, EQ: Eq<Kind2<F, Int, Int>>): List<Law> =
+    listOf(Law("Bitraverse Laws: Identity") { BT.identityBitraverse(BT, cf, EQ) })
```

Comment:
As Bitraverse implies Bifoldable, these tests are: `BifoldableLaws + listOf(...)`

See Foldable and Traverse Laws.

Reply:
ok, I will take a look.